tests/e2e/configs/inversify.config.ts

```diff
@@ -67,22 +60,21 @@ e2eContainer.bind<DriverHelper>(CLASSES.DriverHelper).to(DriverHelper);
 e2eContainer.bind<Dashboard>(CLASSES.Dashboard).to(Dashboard);
 e2eContainer.bind<Workspaces>(CLASSES.Workspaces).to(Workspaces);
 e2eContainer.bind<WorkspaceDetails>(CLASSES.WorkspaceDetails).to(WorkspaceDetails);
-e2eContainer.bind<WorkspaceDetailsPlugins>(CLASSES.WorkspaceDetailsPlugins).to(WorkspaceDetailsPlugins);
 e2eContainer.bind<ScreenCatcher>(CLASSES.ScreenCatcher).to(ScreenCatcher);
 e2eContainer.bind<OcpLoginPage>(CLASSES.OcpLoginPage).to(OcpLoginPage);
 e2eContainer.bind<CheLoginPage>(CLASSES.CheLoginPage).to(CheLoginPage);
 e2eContainer.bind<CheApiRequestHandler>(CLASSES.CheApiRequestHandler).to(CheApiRequestHandler);
-e2eContainer.bind<CheGitApi>(CLASSES.CheGitApi).to(CheGitApi);
-e2eContainer.bind<GitHubUtil>(CLASSES.GitHubUtil).to(GitHubUtil);
 e2eContainer.bind<CreateWorkspace>(CLASSES.CreateWorkspace).to(CreateWorkspace);
 e2eContainer.bind<UpdateAccountInformationPage>(CLASSES.UpdateAccountInformationPage).to(UpdateAccountInformationPage);
 e2eContainer.bind<ProjectAndFileTests>(CLASSES.ProjectAndFileTests).to(ProjectAndFileTests);
 e2eContainer.bind<LoginTests>(CLASSES.LoginTests).to(LoginTests);
-e2eContainer.bind<WorkspaceNameHandler>(CLASSES.WorkspaceNameHandler).to(WorkspaceNameHandler);
-e2eContainer.bind<GitLoginPage>(CLASSES.GitLoginPage).to(GitLoginPage);
-e2eContainer.bind<GitOauthAppsSettings>(CLASSES.GitOauthAppsSettings).to(GitOauthAppsSettings);
 e2eContainer.bind<Sanitizer>(CLASSES.Sanitizer).to(Sanitizer);
 e2eContainer.bind<ApiUrlResolver>(CLASSES.ApiUrlResolver).to(ApiUrlResolver);
 e2eContainer.bind<WorkspaceHandlingTests>(CLASSES.WorkspaceHandlingTests).to(WorkspaceHandlingTests);
 
+if (TestConstants.TS_OCP_LOGIN_PAGE_PROVIDER_TITLE === 'DevSandbox') {
```

Comment:
Would it be simpler to get rid of `ICheLoginPage` and use `RedHatLoginPage` only?

Reply:
As far as I know, these tests are run against OCP only, including:
- Eclipse Che deployed to dogfooding instance https://che-dogfooding.apps.che-dev.x6e0.p1.openshiftapps.com/
- Eclipse Che deployed to OpenShift.CI
- Dev Spaces on test OCP instances
- Dev Spaces on DevSandbox staging
- Dev Spaces on DevSandbox production

Reply:
@ScrewTSW could you help me to run tests on DevSandbox and dogfooding locally, please?

Reply:
It requires container image [quay.io/eclipse/che-e2e](http://quay.io/eclipse/che-e2e) built from this PR.

Reply:
thank you, done

Reply:
Here the test build https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/DevSandbox/job/sandbox-test/5708/

Reply:
✅ Successful run on DevSandbox: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/DevSandbox/job/sandbox-test/5716/console

Reply:
And there is successful run against downstream Dev Spaces: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/7294/